pinot-query-planner/src/test/resources/queries/Aggregates.json

```diff
@@ -0,0 +1,34 @@
+{
+  "aggregates_planning_tests": {
```

Comment:
(nit) If we are planning to keep query planning v/s query execution test resource files separately, then we may want to rename this file to AggregatePlans.json or something similar.

Alternatively, we can keep planning and execution tests in the same file then we can probably restructure this

```
"queries": {
     "planning_tests" : {
      .........
      }
      "execution_tests": {
      ..........
     }
}
```

In any case, need not necessarily do in this PR since we want to do the runtime work soon.

Reply:
I'll rename the files to include "Plans" in the name for now. I kept the planning and execution tests separate since the test folders are in separate modules. Perhaps in the next PR we can create a common tests folder outside the two modules and combine these tests into a single file.

Reply:
done renamed them